### ISSUE TITLE
No feedback if document is currently compiling

### ISSUE THREAD

**Issue by ernstblecha** [alexandervdm/gummi#34]
Especially with bigger files compilation time tends to go up. 

It would be nice if the preview-icon could tell me if currently a compilation is being executed.

This could be achieved by adding a call to 
`previewgui_update_statuslight("gtk-refresh");` 
after line 198 in src/motion.c 
`editortext = latex_update_workfile (latex, editor);`

**Comment by quanticdev** [comment]
I agree, that feature should be easy to implement and would be very helpful!

**Comment by ernstblecha** [comment]
The status icon has been removed, so this is no longer easily fixable. Status indication would still be nice.
